### PR TITLE
BUG: B Mode texture copy wan't using the correct row pitch

### PR DESCRIPTION
**Pull request by sjh26**

### CODE REVIEW

File: src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.cxx [hunk 1]
```diff
@@ -404,10 +404,10 @@ void vtkPlusWinProbeVideoSource::FrameCallback(int length, char* data, char* hHe
         int slicePitch;
         int rowPitch;
         int tLength = WPDXGetFusedTexData(&texture, &slicePitch, &rowPitch);
-        assert(tLength == frameSize[0] * frameSize[1]);
+        //assert(tLength == frameSize[0] * frameSize[1]);
```
Comment (jamesobutler) [inline src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.cxx]: If this line was uncommented, would it still make sense? Or is this obsolete code that can be removed?

Comment (lassoan) [reply src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.cxx]: Yes, if it is important to check this then add proper error handling (log message, return with error, etc.), if not important then just remove the comment.

Comment (sjh26) [reply src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.cxx]: Right, good catch.  The assert as is no longer makes sense, as the texture returned from the API is no longer guaranteed to have the same size as the image frame.  The updated check would be:  tLength == frameSize[0] * rowPitch.  Will put in proper error handling for this and push another commit.

Comment (RMR54) [reply src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.cxx]: Tested recent assert update with hardware, LGTM

**Pull request merged**